### ISSUE TITLE
v0.31.1: SVG input can crash when no fonts are available

### ISSUE THREAD

**Issue by coding-bunny** [lovell/sharp#3393]
### Is this a possible bug in a feature of sharp, unrelated to installation?

- [X] Running `npm install sharp` completes without error.
- [X] Running `node -e "require('sharp')"` completes without error.

### Are you using the latest version of sharp?

- [X] I am using the latest version of `sharp` as reported by `npm view sharp dist-tags.latest`.

### What is the output of running `npx envinfo --binaries --system --npmPackages=sharp --npmGlobalPackages=sharp`?
```
  System:
    OS: Linux 5.13 Ubuntu 20.04.5 LTS (Focal Fossa)
    CPU: (36) x64 Intel(R) Xeon(R) Platinum 8124M CPU @ 3.00GHz
    Memory: 49.88 GB / 68.58 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 16.17.1 - /usr/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 8.19.2 - /usr/bin/npm
  npmPackages:
    sharp: ^0.31.1 => 0.31.1 
```

### What are the steps to reproduce?

Basically, upgrade from 0.31.0 to 0.31.1
Our tests don't work anymore on the CI system because of the following error:

```
root@b05c4513f4f8:~/project# npm run test

> svg_rasterizer_lambda@1.2.9 test
> jest

Fontconfig error: Cannot load default config file: No such file: (null)
Segmentation fault (core dumped)
```

### What is the expected behaviour?

The tests to work as before
Is there some hidden dependency that's suddenly required?
When downgrading to 0.31.0, everything just works

**Comment by lovell** [comment]
Hi, there's been no change to fontconfig between 0.31.0 and 0.31.1, and that message looks like a warning rather than an error.

The mention of `svg_rasterizer_lambda` suggests this might relate to SVG rendering. librsvg was updated from 2.54.5 to 2.55.1, so that's a possible clue.

Are you able to narrow down what causes this, e.g. a single test scenario? We'll need some way to reliably reproduce.

**Comment by coding-bunny** [comment]
After digging around a little bit, I think the underlying problem might actually be related to Python and Ubuntu 20.04 specifically.
Because I upgraded the container image to use Ubuntu Jammy (22.04) and suddenly everything works.
So whatever is used on the 20.04 release is causing problems, cause even downgrading on the CI system resulted in the same problem.

Going to close this for now as I don't think the issue is `sharp`

**Comment by kleisauke** [comment]
It looks like an upstream issue in Pango introduced in version 1.50.10, see: https://gitlab.gnome.org/GNOME/pango/-/issues/701. Cherry-picking MR https://gitlab.gnome.org/GNOME/pango/-/merge_requests/640 would probably resolve this.

**Comment by coding-bunny** [comment]
oh nice find!
Will wait then till that gets fixed before pushing the upgrade on our side, just to play it safe

**Comment by lovell** [comment]
Thanks Kleis, great sleuthing. It looks like the pango maintainers have also added a test case to help prevent this. I'll reopen until the next release that includes the upstream fix.

**Comment by coding-bunny** [comment]
Seems they merged the fixes.
Going to keep our project on Ubuntu 22.04 for now since there it all works/builds fine.
Ubuntu 20.04 might take a while for the fix to trickle down I suppose.

**Comment by wchaws** [comment]
same issue in node alpine docker image

**Comment by lovell** [comment]
sharp v0.31.2 now available with pango v1.50.11
